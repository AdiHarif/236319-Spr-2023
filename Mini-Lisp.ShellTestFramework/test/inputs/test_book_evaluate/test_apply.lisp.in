(apply (nlambda (x) x) '(a) '(a.T)  )
(apply (nlambda (x y) y) '(a b) '(a.T b.T c.T))
(apply (nlambda (x y z) z) '(a b c) '(a.T b.T c.T))
(apply (lambda (x) x) '(a) '(a.T b.T c.T))
(apply (lambda (x y) y) '(a b) '(a.T b.T c.T))
(apply (lambda (x y z) z) '(a b c) '(a.T b.T c.T))
(apply (lambda (x) x) '(a) '(a.NIL b.T c.T))
(apply (lambda (x y) y) '(a b) '(a.T b.NIL c.T))
(apply (lambda (x y z) z) '(a b c) '(a.T b.T c.NIL))
