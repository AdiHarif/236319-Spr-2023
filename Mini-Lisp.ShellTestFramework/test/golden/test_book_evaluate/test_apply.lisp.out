A
B
C
T
T
T
NIL
NIL
NIL
